workers/main.py

```
@@ -64,6 +67,73 @@ def handle_cors_preflight(origin):
 
+# ===================================
+# Security Helpers
+# ===================================
+
+JWT_SECRET = "dev-secret-key" # In production, set this via wrangler secret put
+
+def hash_password(password, salt=None):
+    """Hash a password using PBKDF2"""
+    if salt is None:
+        import random
+        salt = "".join([random.choice("0123456789abcdef") for _ in range(16)])
```

Ruff (0.15.4)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)